docs/user_guides/single_table/constraints.rst

```diff
@@ -184,12 +184,50 @@ constraint by passing it:
         handling_strategy='transform'
     )
 
+OneHotEncoding Constraint
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another constraint available is the ``OneHotEncoding`` constraint.
+This constraint allows the user to specify a list of columns where each row 
+is a one hot vector. Then, the constraint will make sure that the output
+of the model is transformed so that the column with the largest value is
+set to 1 while all other columns are set to 0. 
+
+Since none of the columns of the demo dataset satisfy the requirements for this
```

Comment:
Rather than this, maybe it would be interesting to add a few one-hot-encoded columns to the demo data?

Reply:
Done.